docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Expose workloads in multiple namespaces with a single APIRule definition
+---
+
+This tutorial shows how to expose service endpoints in multiple namespaces using API Gateway Controller.
+   > **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [OAuth2](./apix-05-expose-and-secure-workload-oauth2.md) or [JWT](./apix-08-expose-and-secure-workload-jwt.md).
+
+
+## Expose and access your workloads in multiple namespaces
+
+Follow the instructions to expose and access your unsecured instance of the HttpBin service and unsecured sample Function.
+
+1. Create a Namespace for the HttpBin service and export its value as an environment variable. Run:
+
+   ```bash
+   export NAMESPACE_HTTPBIN={NAMESPACE_HTTPBIN}
+   kubectl create ns $NAMESPACE_HTTPBIN
+   kubectl label namespace $NAMESPACE_HTTPBIN istio-injection=enabled --overwrite
+   ```
+
+2. Create a different Namespace for the Function service and export its value as an environment variable. Run:
+
+   ```bash
+   export NAMESPACE_FUNCTION={NAMESPACE_FUNCTION}
+   kubectl create ns $NAMESPACE_FUNCTION
+   kubectl label namespace $NAMESPACE_FUNCTION istio-injection=enabled --overwrite
+   ```
+
+3. Deploy an instance of the HttpBin service in its Namespace:
+
+   ```bash
+   kubectl -n $NAMESPACE_HTTPBIN create -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+   ```
+
+4. Create a Function using the [supplied code](./assets/function.yaml) in its Namespace:
+
+   ```bash
+   kubectl -n $NAMESPACE_FUNCTION apply -f https://raw.githubusercontent.com/kyma-project/kyma/main/docs/03-tutorials/assets/function.yaml
+   ```
+
+## Next steps
+
+1. Create a Namespace for the Gateway and APIRule CRs. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE}
+   kubectl create ns $NAMESPACE
+   kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+   ```
+
+   >**NOTE:** Skip this step if you already have a Namespace.
+
+2. Export the following values as environment variables:
+
+   ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE_APIRULE/httpbin-gateway
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com. If you don't want to use your custom domain, replace `DOMAIN_NAME` with a Kyma domain and `$NAMESPACE/httpbin-gateway` with Kyma's default Gateway `kyma-system/kyma-gateway`
+
+3. Expose the HttpBin service and the Function service by creating an APIRule CR in their respective Namespaces. Run:
```

Comment:
APIRule is created in its own namespace

Reply:
Please see if it is now more understandable.